Fastlane sign for fetching provisioning profiles is not working: `block in fetch_profiles': undefined method `identifier' for nil:NilClass (NoMethodError)

<!-- Thanks for helping fastlane! Before you submit your issue, please make sure you followed our checklist and check the appropriate boxes by putting an x in the [ ]: [x] -->

### New Issue Checklist

- [ x ] Updated fastlane to the latest version
- [ x ] I read the [Contribution Guidelines](https://github.com/fastlane/fastlane/blob/master/CONTRIBUTING.md)
- [ x ] I read [docs.fastlane.tools](https://docs.fastlane.tools)
- [ x ] I searched for [existing GitHub issues](https://github.com/fastlane/fastlane/issues)

### Issue Description
With running following command 
fastlane sigh --app_identifier $APP_ID --username $FASTLANE_USER --provisioning_name $PROVISIONINF_NAME --team_id $TEAM_ID --template_name "$TEMPLATE_NAME" --ignore_profiles_with_different_name --skip_certificate_verification

following output is generated:
[12:50:05]: Fetching profiles...

Looking for related GitHub issues on fastlane/fastlane...

➡️  when I run cordova run , it have a error
    https://github.com/fastlane/fastlane/issues/7467 [closed] 2 💬
    14 May 2017

➡️  "Could not find app with app identifier" when init Mac project
    https://github.com/fastlane/fastlane/issues/7480 [closed] 5 💬
    22 Mar 2017

➡️  How can I turnoff the password re-enter and just get the error when the password is incorrect?
    https://github.com/fastlane/fastlane/issues/7484 [closed] 5 💬
    24 May 2017

and 19624 more at: [https://github.com/fastlane/fastlane/search?q=undefined%20method%20%60identifier%27%20for%20nil%0A%0A%20%20%20%20%20%20%20%20profile.bundle_id.identifier%20%3D%3D%20Sigh.config%5B%3Aapp_identifier%5D%0A%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%5E%5E%5E%5E%5E%5E%5E%5E%5E%5E%5E&type=Issues&utf8=✓](https://github.com/fastlane/fastlane/search?q=undefined%20method%20%60identifier%27%20for%20nil%0A%0A%20%20%20%20%20%20%20%20profile.bundle_id.identifier%20%3D%3D%20Sigh.config%5B%3Aapp_identifier%5D%0A%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%5E%5E%5E%5E%5E%5E%5E%5E%5E%5E%5E&type=Issues&utf8=%E2%9C%93)

🔗  You can ⌘ + double-click on links to open them directly in your browser.
/opt/homebrew/Cellar/fastlane/2.212.2/libexec/gems/fastlane-2.212.2/sigh/lib/sigh/runner.rb:105:in `block in fetch_profiles': \e[31m[!] undefined method `identifier' for nil:NilClass (NoMethodError)

        profile.bundle_id.identifier == Sigh.config[:app_identifier]
                         ^^^^^^^^^^^\e[0m
	from /opt/homebrew/Cellar/fastlane/2.212.2/libexec/gems/fastlane-2.212.2/sigh/lib/sigh/runner.rb:104:in `select'
	from /opt/homebrew/Cellar/fastlane/2.212.2/libexec/gems/fastlane-2.212.2/sigh/lib/sigh/runner.rb:104:in `fetch_profiles'
	from /opt/homebrew/Cellar/fastlane/2.212.2/libexec/gems/fastlane-2.212.2/sigh/lib/sigh/runner.rb:38:in `run'
	from /opt/homebrew/Cellar/fastlane/2.212.2/libexec/gems/fastlane-2.212.2/sigh/lib/sigh/manager.rb:8:in `start'
	from /opt/homebrew/Cellar/fastlane/2.212.2/libexec/gems/fastlane-2.212.2/sigh/lib/sigh/commands_generator.rb:62:in `block (2 levels) in run'
	from /opt/homebrew/Cellar/fastlane/2.212.2/libexec/gems/commander-4.6.0/lib/commander/command.rb:187:in `call'
	from /opt/homebrew/Cellar/fastlane/2.212.2/libexec/gems/commander-4.6.0/lib/commander/command.rb:157:in `run'
	from /opt/homebrew/Cellar/fastlane/2.212.2/libexec/gems/commander-4.6.0/lib/commander/runner.rb:444:in `run_active_command'
	from /opt/homebrew/Cellar/fastlane/2.212.2/libexec/gems/fastlane-2.212.2/fastlane_core/lib/fastlane_core/ui/fastlane_runner.rb:124:in `run!'
	from /opt/homebrew/Cellar/fastlane/2.212.2/libexec/gems/commander-4.6.0/lib/commander/delegates.rb:18:in `run!'
	from /opt/homebrew/Cellar/fastlane/2.212.2/libexec/gems/fastlane-2.212.2/sigh/lib/sigh/commands_generator.rb:140:in `run'
	from /opt/homebrew/Cellar/fastlane/2.212.2/libexec/gems/fastlane-2.212.2/sigh/lib/sigh/commands_generator.rb:17:in `start'
	from /opt/homebrew/Cellar/fastlane/2.212.2/libexec/gems/fastlane-2.212.2/fastlane/lib/fastlane/cli_tools_distributor.rb:115:in `take_off'
	from /opt/homebrew/Cellar/fastlane/2.212.2/libexec/gems/fastlane-2.212.2/bin/fastlane:23:in `<top (required)>'
	from /opt/homebrew/Cellar/fastlane/2.212.2/libexec/bin/fastlane:25:in `load'
	from /opt/homebrew/Cellar/fastlane/2.212.2/libexec/bin/fastlane:25:in `<main>'
/opt/homebrew/Cellar/fastlane/2.212.2/libexec/gems/fastlane-2.212.2/sigh/lib/sigh/runner.rb:105:in `block in fetch_profiles': undefined method `identifier' for nil:NilClass (NoMethodError)

        profile.bundle_id.identifier == Sigh.config[:app_identifier]
                         ^^^^^^^^^^^
	from /opt/homebrew/Cellar/fastlane/2.212.2/libexec/gems/fastlane-2.212.2/sigh/lib/sigh/runner.rb:104:in `select'
	from /opt/homebrew/Cellar/fastlane/2.212.2/libexec/gems/fastlane-2.212.2/sigh/lib/sigh/runner.rb:104:in `fetch_profiles'
	from /opt/homebrew/Cellar/fastlane/2.212.2/libexec/gems/fastlane-2.212.2/sigh/lib/sigh/runner.rb:38:in `run'
	from /opt/homebrew/Cellar/fastlane/2.212.2/libexec/gems/fastlane-2.212.2/sigh/lib/sigh/manager.rb:8:in `start'
	from /opt/homebrew/Cellar/fastlane/2.212.2/libexec/gems/fastlane-2.212.2/sigh/lib/sigh/commands_generator.rb:62:in `block (2 levels) in run'
	from /opt/homebrew/Cellar/fastlane/2.212.2/libexec/gems/commander-4.6.0/lib/commander/command.rb:187:in `call'
	from /opt/homebrew/Cellar/fastlane/2.212.2/libexec/gems/commander-4.6.0/lib/commander/command.rb:157:in `run'
	from /opt/homebrew/Cellar/fastlane/2.212.2/libexec/gems/commander-4.6.0/lib/commander/runner.rb:444:in `run_active_command'
	from /opt/homebrew/Cellar/fastlane/2.212.2/libexec/gems/fastlane-2.212.2/fastlane_core/lib/fastlane_core/ui/fastlane_runner.rb:124:in `run!'
	from /opt/homebrew/Cellar/fastlane/2.212.2/libexec/gems/commander-4.6.0/lib/commander/delegates.rb:18:in `run!'
	from /opt/homebrew/Cellar/fastlane/2.212.2/libexec/gems/fastlane-2.212.2/sigh/lib/sigh/commands_generator.rb:140:in `run'
	from /opt/homebrew/Cellar/fastlane/2.212.2/libexec/gems/fastlane-2.212.2/sigh/lib/sigh/commands_generator.rb:17:in `start'
	from /opt/homebrew/Cellar/fastlane/2.212.2/libexec/gems/fastlane-2.212.2/fastlane/lib/fastlane/cli_tools_distributor.rb:115:in `take_off'
	from /opt/homebrew/Cellar/fastlane/2.212.2/libexec/gems/fastlane-2.212.2/bin/fastlane:23:in `<top (required)>'
	from /opt/homebrew/Cellar/fastlane/2.212.2/libexec/bin/fastlane:25:in `load'
	from /opt/homebrew/Cellar/fastlane/2.212.2/libexec/bin/fastlane:25:in `<main>'
Error: Process completed with exit code 1.

##### Command executed
fastlane sigh --app_identifier $APP_ID --username $FASTLANE_USER --provisioning_name $PROVISIONINF_NAME --team_id $TEAM_ID --template_name "$TEMPLATE_NAME" --ignore_profiles_with_different_name --skip_certificate_verification
y
### Environment

<!-- 
Please run `fastlane env` and copy the output below. This will help us help you.
If you used the `--capture_output` option, please remove this block as it is already included there. 
-->

<details>
<details><summary>✅ fastlane environment ✅</summary>

### Stack

| Key                         | Value                                       |
| --------------------------- | ------------------------------------------- |
| OS                          | 13.3.1                                      |
| Ruby                        | 2.7.7                                       |
| Bundler?                    | false                                       |
| Git                         | git version 2.37.1 (Apple Git-137.1)        |
| Installation Source         | /usr/local/bin/fastlane                     |
| Host                        | macOS 13.3.1 (22E261)                       |
| Ruby Lib Dir                | /usr/local/Cellar/ruby@2.7/2.7.7/lib        |
| OpenSSL Version             | OpenSSL 1.1.1s  1 Nov 2022                  |
| Is contained                | false                                       |
| Is homebrew                 | false                                       |
| Is installed via Fabric.app | false                                       |
| Xcode Path                  | /Applications/Xcode.app/Contents/Developer/ |
| Xcode Version               | 14.2                                        |
| Swift Version               | 5.7.2                                       |


### System Locale

| Variable | Value       |   |
| -------- | ----------- | - |
| LANG     | de_DE.UTF-8 | ✅ |
| LC_ALL   |             |   |
| LANGUAGE |             |   |


### fastlane files:

**No Fastfile found**


**No Appfile found**


### fastlane gems

| Gem      | Version | Update-Status  |
| -------- | ------- | -------------- |
| fastlane | 2.212.1 | 💥 Check failed |


</details>
</details>

--- a/sigh/lib/sigh/runner.rb
+++ b/sigh/lib/sigh/runner.rb
@@ -91,7 +91,7 @@ module Sigh
       results = Sigh.config[:cached_profiles]
       results ||= Spaceship::ConnectAPI::Profile.all(filter: filter, includes: includes)
       results.select! do |profile|
-        profile.bundle_id.identifier == Sigh.config[:app_identifier]
+        profile.bundle_id&.identifier == Sigh.config[:app_identifier]
       end
 
       results = results.find_all do |current_profile|

